apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -30,18 +26,35 @@ describe('Button Accessibility Testing', () => {
     await ButtonPageObject.scrollToTestElement();
   });
 
-  it('Button - Validate accessibilityRole is correct', async () => {
-    await expect(await ButtonPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+  it('Validate "accessibilityRole" defaults to Button "ControlType" element attribute.', async () => {
+    await expect(
+      await ButtonPageObject.compareAttribute(ButtonPageObject._primaryComponent, Attribute.AccessibilityRole, BUTTON_A11Y_ROLE),
+    ).toBeTrue();
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Button - Set accessibilityLabel', async () => {
-    await expect(await ButtonPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(BUTTON_ACCESSIBILITY_LABEL_DEPRECATED);
+  it('Set "accessibilityLabel" prop. Validate "accessibilityLabel" value propagates to "Name" element attribute.', async () => {
+    await expect(
+      await ButtonPageObject.compareAttribute(
+        ButtonPageObject._primaryComponent,
+        Attribute.AccessibilityLabel,
+        BUTTON_ACCESSIBILITY_LABEL_DEPRECATED,
+      ),
+    ).toBeTrue();
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Button - Do not set accessibilityLabel -> Default to Button label', async () => {
-    await expect(await ButtonPageObject.getAccessibilityLabel(ComponentSelector.Secondary)).toEqual(BUTTON_TEST_COMPONENT_LABEL_DEPRECATED);
+  it('Do NOT set "accessibilityLabel" prop. Validate "accessibilityLabel" value defaults to the button label.', async () => {
```

Comment:
Should we say "Validate the "Name" element attribute value defaults to the button label"?

Reply:
Yes we should.